context/jest.config.js

```diff
@@ -1,9 +1,16 @@
 module.exports = {
   testPathIgnorePatterns: ['jest.config.js', '/node_modules/', '/cypress/'],
-  setupFilesAfterEnv: ['./setuptests.js'],
+  setupFilesAfterEnv: ['<rootDir>/test-utils/setupTests.js'],
   moduleNameMapper: {
+    '^.+\\.css$': '<rootDir>/test-utils/__mocks__/style.js',
     '^components(.*)$': '<rootDir>/app/static/js/components$1',
     '^static(.*)$': '<rootDir>/app/static/$1',
-    '^test-utils.js': '<rootDir>/test-utils.js',
+    '^test-utils(.*)$': '<rootDir>/test-utils$1',
+    'search-schema-definitions$': '<rootDir>/search-schema/data/definitions.yaml',
+    'metadata-field-descriptions$': '<rootDir>/ingest-validation-tools/docs/field-descriptions.yaml',
+  },
+  transform: {
+    '^.+\\.[t|j]sx?$': 'babel-jest',
```

Comment:
Are we already able to support typescript, or would that require something else?

Reply:
No that would require more configuration throughout the application. I can remove the `t` to avoid confusion?

Reply:
unless we can actually handle it, let's remove it.